src/apps/search/panels/BasePanel.tsx

```diff
@@ -336,6 +354,9 @@ const BasePanel = (props: Props) => {
         onClose={() => navigate('/')}
         relations={relations()}
         title={name}
+        detailPageUrl={config.detail_pages.includes(plurals[props.name]) 
```

Comment:
Would it make sense to do this logic in the model-specific implementations of the component and pass the URL as a prop?